src/rosdiscover/cli.py

```diff
@@ -166,10 +171,14 @@ def _periodic_observe(interval: float, args: argparse.Namespace) -> SystemSummar
         if process and not process.poll():
             logger.warning(f"Observations seemed to have stopped before '{args.run_script}' completed.")
 
-        logger.info(f"Finished observing - {iterations+1} observations in total after {stopwatch.duration} seconds.")
+        logger.info(f"Finished observing - {iterations+1} observations in total after {stopwatch.duration:.3g} "
```

Comment:
why `g` (which is sometimes a lowercase exponential format) rather than `f`?

Reply:
typo

Reply:
Which funnily enough produces the same output in this case.

Reply:
Done